README.md

```diff
@@ -42,6 +42,30 @@ This project is based on [Open Camera](https://opencamera.org.uk/)  —  a popul
     - *Note: phase, which is returned by* ```start_recording``` *method, can be used to perform synchronization with external devices*
  ![remote control methods](https://www.websequencediagrams.com/files/render?link=6txhpHrdgaebT4DYz2C3SaEQjHM1esYDkJZJvPZcgCJHbRAg3c8hqcJYgOmGirze)
 
+### Synchronized recording on multiple smartphones (RecSync)
+
+**Important**: smartphones are required to support real-time timestamping to be correctly synchronized. This can be checked on the preview message when RecSync is enabled ("Timestamp source" should be "realtime").
+![screenshot timestamp source](https://i.imgur.com/vQHufyV.png)
+
+**Leader smartphone setup:**
+- Start a **Wi-Fi hotspot**
+- Open OpenCamera Sensors, go to preferences - "RecSync settings..." and enable the **"Use RecSync"** switch
+- (Optional) Enable **phase alignment** option if synchronization precision better than half of a frame duration is required
+- (Optional) Choose which camera settings will be broadcasted to client smartphones in the **"Sync settings"** section
+- Switch to video, adjust the camera settings as needed and press the **settings synchronization button**
+- Wait for client smartphones to connect if needed
+- (Optional) If phase alignment was enabled, press the **phase alignment button** to start the alignment and wait for it to finish (green "Phase error" text on the preview indicates the success of the alignment)
```

Comment:
Phase error on the screen -> but all is ok)) Maybe it is better to change the text to success?

Reply:
In this context "error" means "shift from an expected value", which is considered ok when its value doesn't exceed a predetermined threshold. If the threshold is not exceeded, the text line on the preview, where the shift is specified (it is called "Phase error") is green, otherwise it is red.
So, I think it is correct to call this shift a "error", because it is one. When it is not a "serious error" and can be considered an aligned state, the text indicating it is green -- that's what is written here.
I'll think of a way to paraphrase this.

Reply:
Made this clearer now